config/kubernetes-sigs/org.yaml

```diff
@@ -557,6 +557,7 @@ members:
 - mrunalp
 - msau42
 - mskanth972
+- seanzatzdev-amazon
```

Comment:
should be in alphabetical order . i think the unit test will complain.

Reply:
you don't need this @seanzatzdev-amazon - there's an entry already in the file :)

Reply:
ref ^^^ https://github.com/kubernetes/org/pull/4858/files#diff-0952ca7c15fba08fc29368a53a6b91f1f2e8a33aae5022f25a890f5be8335b8bR739

Reply:
Ah oops, thanks for pointing that out!